build.gradle

```diff
@@ -287,6 +293,8 @@ task shadowTestJar(type: ShadowJar) {
     from(project.sourceSets.main.output, project.sourceSets.test.output)
     configurations = [project.configurations.testRuntime]
     dependencies {
+        include(dependency('org.slf4j:slf4j-api:1.7.25'))
```

Comment:
@danking @tpoterba Can you comment on this?  I don't quite remember how the logging issue from last week was resolved.